src/Nethermind/Nethermind.Monitoring/Metrics/MetricsManualNamedAttribute.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace Nethermind.Monitoring.Metrics;
+
+[AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
+public class MetricsManualNamedAttribute : Attribute
+{
+    public string Name { get; }
+
+    /// <summary>
+    /// Collects a user defined name for use in metrics registration, will be used directly as given (not nethermind_var_name_snake_case form CamelCase without nethermind)
+    /// </summary>
+    /// <param name="metricName">Metrics name</param>
+    public MetricsManualNamedAttribute(string metricName)
```

Comment:
IMO not worth creating custom attribute, better to reuse built-in `DisplayNameAttribute` that was created just for that and is used commonly, like in serialization libraries.

Reply:
`DisplayNameAttribute` for properties only, and we often have stuff that needs to be `ref`ed. So will go with `DataMemberAttribute.Name`